declarations/WebpackOptions.d.ts

```diff
@@ -2929,6 +2929,24 @@ export interface JavascriptParserOptions {
 	 * Enable/disable parsing of magic comments in CommonJs syntax.
 	 */
 	commonjsMagicComments?: boolean;
+	/**
+	 * Specifies global mode for dynamic import.
+	 */
+	dynamicImportMode?:
+		| "sync"
+		| "eager"
+		| "weak"
+		| "async-weak"
+		| "lazy"
+		| "lazy-once";
+	/**
+	 * Specifies global prefetch for dynamic import.
+	 */
+	dynamicImportPrefetch?: boolean;
```

Comment:
Maybe these should be `boolean | number` to allow to specify a default priority.